src/sdsAccess.ts

```diff
@@ -330,18 +330,49 @@ export async function getScriptNamesFromServer(sdsConnection: SDSConnection, par
 
 
 
-export async function getScriptParameters(sdsConnection: SDSConnection, params: any[]): Promise<any[]> {
+export async function getScriptParameters(sdsConnection: SDSConnection, params: scriptT[]): Promise<string[]> {
     return new Promise<any[]>((resolve, reject) => {
-        let jsonIn = '{\n"nameScript":"test_VSCode_folder1.1"\n}';
-        sdsConnection.callClassOperation('PortalScript.*generic', ['getScriptParameters2', jsonIn]).then((param) => {
-            resolve(param);
+        let scriptname: string = params[0].name;
+        let jsonIn: string = '{\n"nameScript":"' + scriptname + '"\n}';
```

Comment:
You can use `const` instead of `let` keyword for both, `scriptname` and `jsonIn`.